src/components/application_manager/CMakeLists.txt

```diff
@@ -114,24 +114,6 @@ file (GLOB MOBILE_COMMANDS_SOURCES
 )
 
 if (NOT REMOTE_CONTROL)    
-    list(REMOVE_ITEM MOBILE_COMMANDS_SOURCES
-        ${COMMANDS_SOURCE_DIR}/mobile/set_interior_vehicle_data_request.cc)
-    list(REMOVE_ITEM MOBILE_COMMANDS_SOURCES
-        ${COMMANDS_SOURCE_DIR}/mobile/get_interior_vehicle_data_response.cc)
-    list(REMOVE_ITEM MOBILE_COMMANDS_SOURCES
-        ${COMMANDS_SOURCE_DIR}/mobile/get_interior_vehicle_data_capabilities_response.cc)
```

Comment:
@LevchenkoS please make sure that this pull requests will be interdependent to have possibility to restore each one of them independently